pkg/fleets/controller_test.go

```diff
@@ -685,6 +685,130 @@ func TestControllerUpdateFleetPlayerStatus(t *testing.T) {
 	assert.True(t, updated)
 }
 
+// nolint // Linter errors on lines are duplicate of TestControllerUpdateFleetListStatus
```

Comment:
(non blocking) nolint at all, or did you want to only nolint:duplicate (or whatever the duplicate lint tag is?)

Reply:
Good call, I believe it's the dubl tag.